specifications/api/swagger.yaml

```
@@ -1791,6 +1797,10 @@ components:
             With Alonzo, `minimum_utxo_value` is not a real protocol parameter, but rather
             derived from from the Alonzo genesis `adaPerUTxOWord`.
         eras: *ApiEraInfo
+        maximum_collateral_inputs:
```

Another suggestion: (to make clear that it's a maximum count, rather than a maximum that applies to each input itself):
```suggestion
        maximum_collateral_input_count:
```